rollup/internal/controller/relayer/l1_relayer_metrics.go

```diff
@@ -9,7 +9,8 @@ import (
 
 type l1RelayerMetrics struct {
 	rollupL1RelayerGasPriceOraclerRunTotal      prometheus.Counter
-	rollupL1RelayerLastGasPrice                 prometheus.Gauge
+	rollupL1RelayerLastestBaseFee               prometheus.Gauge
```

Comment:
```suggestion
	rollupL1RelayerLatestBaseFee               prometheus.Gauge
```

Reply:
fixed in [23f46af](https://github.com/scroll-tech/scroll/pull/1344/commits/23f46afcb55dae3d8b317dec27d537b9fbd61f33).